### ISSUE TITLE
Community curation

### ISSUE THREAD

**Issue by lnielsen** [zenodo/zenodo#35]
- [ ] Allow multiple curators
- [ ] Allow curators to edit metadata.

**Comment by lnielsen** [comment]
Related to #151

**Comment by lnielsen** [comment]
See use case in #991

**Comment by jvvacano** [comment]
Are there any news regarding the overhaul of Community curation features? i.e. #1710  #1807 #136 #810 #1325 #1010 

Thanks!

**Comment by lnielsen** [comment]
Yes, it's being implemented now. It's part of the InvenioRDM project, that Zenodo will run ontop in autumn 2020. See https://inveniosoftware.org/products/rdm/ and https://about.zenodo.org/roadmap/

**Comment by sje30** [comment]
Yes, we would like multiple curators for a community so look forward to this update.  CC: @nuest

**Comment by dagewa** [comment]
Hi folks, is there any progress on this? I would like to share the responsibility for responding to "A record was requested to be added to your community" emails with other curators. Thanks!

**Comment by moble** [comment]
@dagewa #810 is a more recent version of this issue.  The schedule has slipped to autumn 2021 now, as mentioned in [this comment](https://github.com/zenodo/zenodo/issues/810#issuecomment-753917462).  (And in fairness, 2020 has been rough on my schedules too.)
